Project/src/main/java/com/github/lgooddatepicker/components/DatePickerSettings.java

```diff
@@ -1182,6 +1189,11 @@ public boolean getWeekNumbersDisplayed() {
   public boolean getWeekNumbersWillOverrideFirstDayOfWeek() {
     return weekNumbersWillOverrideFirstDayOfWeek;
   }
+  
+  /** getYearSelectScroll, Returns the value of this setting (if to show up/down arrow on year select pop up) */
```

Comment:
seems inconsistent formatting;  old line 803 gets reformatted to new line 808 but this line (1193) is formatted like the old line 803,  How did this happen?

Reply:
Sorry, it is based on my local version before the bot applied formatting on your version (had to restore local history after a merge/rebase (lost my changes because noob with github/pull requests), so I killed some of the cleanup by the bot with that).
Other issues are cause by that as well.

The bot would reapply the cleanup if it runs again. Our should I create a new pull request that is clean?